code/modules/projectiles/ammunition/ammo_boxes.dm

```diff
@@ -18,55 +18,62 @@
 
 /obj/item/ammo_box/c9mm
 	name = "ammo box (9mm)"
+	desc = "An ammunition box filled with 9mm pistol cartridges, commonly used in handguns and submachine guns."
```

Comment:
```suggestion
	desc = "A pack ammunition box filled with thirty 9mm pistol rounds, commonly used in handguns and submachine guns."
```

And now I am going to add capacity to all the other descriptions because this is based. Also not carts.